### PR TITLE
New case to check disk size increased can be detected automatically

### PR DESCRIPTION
**Pull request by YongxueHong**
New a case that checks the size fo disk increased could
be detected automatically.

ID: 1731617
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: qemu/tests/block_auto_detect_size_increased.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    img_resize_size = img_params.get('image_resize_size')
+    img_filename = storage.get_image_filename(img_params,
+                                              data_dir.get_data_dir())
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    session = utils_test.qemu.windrv_check_running_verifier(
+            vm.wait_for_login(), vm, test, 'viostor', 300)
+    indexes = utils_disk.get_windows_disks_index(session, img_size)
```
Comment (hereischen) [inline qemu/tests/block_auto_detect_size_increased.py]: `indices` is plural form of index.

Comment (YongxueHong) [reply qemu/tests/block_auto_detect_size_increased.py]: ok, I will update it, thanks.

**Comment by menli820** [comment]
Test pass in -drive mode.
(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (957.31 s) 
(2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased.q35: PASS (427.36 s)

hit a  error  in -blockdev mode
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased.q35: ERROR: QMP command 'block_resize' failed (arguments: {'device': 'drive_stg', 'size': 21474836480}, error message: {'class': 'GenericError', 'desc': 'Cannot find device=drive_stg nor node_name='}) (196.59 s) 
I think it  should use 'node_name' :'drive_stg  instead of  'device': 'drive_stg' in QMP command in blockdev mode.

**Comment by YongxueHong** [comment]
> drive_stg

have updated.

**Comment by YongxueHong** [comment]
> Test pass in -drive mode.
> (1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (957.31 s)
> (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased.q35: PASS (427.36 s)
> 
> hit a error in -blockdev mode
> (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased.q35: ERROR: QMP command 'block_resize' failed (arguments: {'device': 'drive_stg', 'size': 21474836480}, error message: {'class': 'GenericError', 'desc': 'Cannot find device=drive_stg nor node_name='}) (196.59 s)
> I think it should use 'node_name' :'drive_stg instead of 'device': 'drive_stg' in QMP command in blockdev mode.

have updated.

**Comment by menli820** [comment]
Test pass on my side on both -blockdev and -drive mode.
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased.q35: PASS (395.77 s)

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=block_auto_detect_size_increased  --guestname=Win2019 --driveformat=virtio_blk --platform=x86_64
(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (862.35 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_auto_detect_size_increased: PASS (104.87 s)

LGTM

**Pull request merged**